packages/x-data-grid-generator/src/hooks/useMockServer.ts

```diff
@@ -50,7 +50,7 @@ function decodeParams(url: string): GridGetRowsParams {
 
   for (const [key, value] of array) {
     try {
-      decodedParams[key] = JSON.parse(decodeURIComponent(value));
+      decodedParams[key] = JSON.parse(value);
```

Comment:
should this be called `parsedParams` now?

Reply:
Nice suggestion, IMO though, `decodedParams` would still be a better name, since the function `decodeParams` technically returns decoded params. How those are being computed (using the URL constructor now instead of manual decode) is an implementation detail.
It would also contain those values that aren't actually parsed but handled in the catch block, e.g. string values. (e.g. `JSON.parse('some string')` would throw)
What do you think?